src/libs/actions/Report.js

```diff
@@ -243,7 +244,8 @@ function fetchIOUReport(iouReportID, chatReportID) {
         }
         const iouReportData = response.reports[iouReportID];
         if (!iouReportData) {
-            console.error(`No iouReportData found for reportID ${iouReportID}`);
+            // This is occuring due to 'fetchChatReportsByIDs' calling this function with a IOUReportID that has
+            // already been settled. In this case it is expected that no data is returned from 'getIOU'.
```

Comment:
Can we standardize the term "settled" and "paid"? We should call it whatever it is called in auth.

Reply:
Yeah, I've made this change to all references of 'settled'.